server/tables/tables.go

```diff
@@ -578,78 +578,73 @@ func PreAutoMigrate(db *gorm.DB) ([]PostAutoMigrateLogic, error) {
 		})
 	}
 
-	if m.HasTable("Invocations") && m.HasColumn(&Invocation{}, "invocation_uuid") {
-		// TODO(luluz): Disable migration code for now
-		// if db.Dialector.Name() == mysqlDialect {
-		// 		err := backfillInvocationUUIDIfNull(db)
-		//	return postMigrate, err
-		//}
-	}
-
 	// Populate invocation_uuid if the column doesn't exist.
-	if m.HasTable("Invocations") && !m.HasColumn(&Invocation{}, "invocation_uuid") {
+	if m.HasTable("Invocations") && m.HasColumn(&Invocation{}, "invocation_pk") {
 		if db.Dialector.Name() == sqliteDialect {
-			// TODO(luluz): Disable migration code for now
-			//postMigrate = append(postMigrate, func() error {
-			//	return postMigrateInvocationUUIDForSQLite(db)
-			//})
+			postMigrate = append(postMigrate, func() error {
+				return postMigrateInvocationUUIDForSQLite(db)
+			})
 		} else if db.Dialector.Name() == mysqlDialect {
-			// TODO(luluz): Disable migration code for now
-			// postMigrate = append(postMigrate, func() error {
-			// return postMigrateInvocationUUIDForMySQL(db)
-			// })
+			postMigrate = append(postMigrate, func() error {
+				return postMigrateInvocationUUIDForMySQL(db)
+			})
 		} else {
 			log.Warningf("Unsupported sql dialect: %q", db.Dialector.Name())
 		}
 	}
 	return postMigrate, nil
 }
 
-func backfillInvocationUUIDIfNull(db *gorm.DB) error {
-	if err := db.Exec(`
-		UPDATE Invocations SET invocation_uuid = UUID_TO_BIN(invocation_id) 
-		WHERE invocation_uuid IS NULL
-	`).Error; err != nil {
-		return err
+func updateInBatch(db *gorm.DB, baseQuery string, batchSize int64) error {
```

Comment:
nit: let's call this updateInBatches since there will be multiple batches

Reply:
Done